piezo_web_app/PiezoWebApp/run_piezo.py

```diff
@@ -95,6 +98,15 @@ def build_app(container, use_route_stem=False):
     return app
 
 
+@gen.coroutine
+def background_tidy(logger):
+    spark_job_service = CONTAINER['spark_job_service']
+    while True:
+        response = spark_job_service.tidy_jobs()
+        logger.debug(f'Summary of jobs tidied at {datetime.datetime.now()}: {response}')
+        yield gen.sleep(3600)
```

Comment:
should this timing come from a configuration setting?

Reply:
https://github.com/ukaea/piezo/pull/109/commits/ce6e7e835a674932da52a3afe0dad617b87af3b2